BUILD.md

```diff
@@ -0,0 +1,29 @@
+# Build guide
+
+ChroMapper is developed with Unity **2019.3.0f6** in C#.
+
+This GitHub repository comes with the assets and scripts you need to easily open it up in Unity. Feel free to fork the repository if you plan on contributing to ChroMapper in any way. Even though ChroMapper is in closed alpha, if you are able to clone or fork and get the project working on your end, I'll allow you to use it as long as you're making contributions to ChroMapper. The setup is straight format
+
+## Development Environment Setup
+* Clone the project from GitHub to your local work folder
+* Download and install [Unity Hub](https://unity3d.com/get-unity/download)
+* Activate your license within Unity Hub, most people should be eligible for a free Personal license
+* If you want to select your Visual Studio installation location install it upfront, otherwise the installer will put it into the default location
+* Install the correct Unity version mentioned in the header of this page from within Unity Hub in the "Installs" section. The project might not be using the latest release. In this case you should consult the "download archive" (from within Unity Hub). The installer will take care of the required dependencies as well.
+* Add the project in the "Projects" section. Select your main folder you cloned from GitHub.
+
+## Running the project
+* Select "File" -> "Build and Run" within the IDE.
+* Warning "Missing Project ID" will appear. Ignore it and select "Yes". 
+* You can ignore the "Discord RPC error"
+
+## Repository layout
+
+The project works the master / development branch setup.
```

Comment:
Again, looks like there's an error here. Were you looking for `The project works with the master / development branch setup.`?